bittensor/_neuron/text/core_validator/__init__.py

```diff
@@ -126,6 +130,7 @@ def __init__(
             self.config.axon._mock = True
         print ( self.config )
 
+        copy.deepcopy(self.config)
```

Comment:
why are executing deepcopy three times? Do we need that? copy.deepcopy returns the result as a value right? So, why are not receiving the result?

Reply:
It was for some debugging purpose, and it is redundant now.

Reply:
can we remove it now??